internal/data-ingest-cli/README.md

```diff
@@ -33,15 +36,17 @@ go build -o data-ingest
 The tool accepts the following input parameters:
 
 - `--input-file`: The name of the input file to read data from.
-- `--input-format`: The input format of the ingested data (options: `otlpjson`, `syslog`, `statsd`). 
+- `--input-format`: The input format of the ingested data (options: `otlpjson`, `syslog`, `statsd`).
 - `--collector-url`: The URL of the OpenTelemetry collector.
 - `--output-file`: The file in which to store the received data.
 - `--receiver-port`: The port of the OTLP receiver created to act as a sink for the collector.
+- `--receiver-type`: The type of receiver created to act as a sink for the collector (options: `http`, `grpc`).
```

Comment:
should we add a short note about disabling compression in the collector config when using otlphttp?